src/main/java/org/humancellatlas/ingest/project/web/ProjectController.java

```diff
@@ -219,4 +219,8 @@ public ResponseEntity<PagedResources<Resource<Project>>> filterProjects(
         return ResponseEntity.ok(pagedResourcesAssembler.toResource(projects, resourceAssembler));
     }
 
+    @GetMapping(path="projects/{id}/audit-logs")
+    public ResponseEntity<?> projectAuditLog(@PathVariable("id") Project project) {
+        return ResponseEntity.ok(projectService.getProjectAuditLog(project));
```

Comment:
At present it just returns an empty list instead of `404`.  Is that alright? Is there any advantage to explicitly returning a `404` instead of an empty list?